web/app_dev.php

```diff
@@ -10,7 +10,7 @@
 // This check prevents access to debug front controllers that are deployed by accident to production servers.
 // Feel free to remove this, extend it, or make something more sophisticated.
 
-//*
+/*
```

Comment:
I'm 👎  with this change as it may generate a serious security issue if you forget to drop the app_dev.php file. Otherwise adding the environment in the `.env` file looks like a good idea to me.

You'll need to change the `app.php` file to support an environment var.